credentials/apps/verifiable_credentials/migrations/0011_auto_20230405_0915.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.18 on 2023-04-05 09:15
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
Does it possible to still squash this migration with issuanceconfiguration and issuanceline creation migration?
Only if it wasn't merged to upstream master or we won't spend a lot of time on it.

Reply:
I also thought on it... This is something that could be cleaned before the final review.
It won't break us until we'll make the squash during the not synced Dev and Stage (everything we have on the Dev must be pushed to the Stage before migrations are squashed).